prime/src/components/graph/PnLGraph.tsx

```diff
@@ -370,7 +428,7 @@ export default function PnLGraph(props: PnLGraphProps) {
               />
               <YAxis stroke={SECONDARY_COLOR} fontSize='14px' />
               <ReferenceLine y={0} stroke={SECONDARY_COLOR} />
-              <ReferenceLine x={price} stroke={SECONDARY_COLOR} strokeWidth={2} />
+              <ReferenceLine x={currentPrice} stroke={SECONDARY_COLOR} strokeWidth={2} />
```

Comment:
Let's have an axis-colored reference line for `previousPrice`, and a bright white (or blue or something) reference line for `currentPrice`